package.json

```diff
@@ -24,6 +24,7 @@
     "react-dom": "^18.2.0",
     "react-giphy-searchbox": "^1.5.4",
     "react-grid-layout": "^1.3.4",
+    "react-router-dom": "^6.11.2",
```

Comment:
Let's remove this dependency -- I fixed the `LinkBlock` which was pulling it in accidentally, so we shouldn't need `react-router` anymore.